src/game/player.rs

```diff
@@ -12,8 +12,8 @@ pub fn player_plugin(app: &mut App) {
         FixedUpdate,
         (
             control_player,
-            // Uncommend to following line after you've done the exercises 6.4 to have srpites for the ground
-            // gravity
+            // Uncomment to following line after you've done the exercises 6.4
+            // to have sprites for the ground gravity
```

Comment:
```suggestion
            // Uncomment the following line after you've done the exercises 6.4 to have sprites for the ground
            // gravity
```
`gravity` needs to be on its own line as it's the system name, and if the comment is on two lines "the following line" doesn't work anymore